toolchain/lower/testdata/interop/cpp/function_decl.carbon

```diff
@@ -22,6 +22,20 @@ fn MyF() {
   Cpp.foo();
 }
 
+// --- inline_function_decl.h
+
+inline void foo() {}
```

Comment:
Similar to the other test, distinct names and a .h file?

Reply:
See https://github.com/carbon-language/carbon-lang/pull/5406#discussion_r2073664558